src/components/scenes/SendScene2.tsx

```diff
@@ -984,6 +992,13 @@ const SendComponent = (props: Props) => {
           } else {
             e.message = sprintf(lstrings.stake_error_insufficient_s, errorCurrencyCode)
           }
+
+          if (spendInfo.tokenId !== insufficientFunds.tokenId) {
+            // Show the modal if the flip input modal is closed or save it to show when it closes later
+            const modalPromise = async () =>
+              await Airship.show(bridge => <InsufficientFeesModal bridge={bridge} coreError={insufficientFunds} navigation={navigation} wallet={coreWallet} />)
+            flipInputModalOpen.current == null ? (insufficientFeesModalPromise.current = modalPromise) : modalPromise().catch(error => showError(error))
```

Comment:
This condition will never be true, because it's always `true | false`, not `null`.